examples/ex18p.cpp

```diff
@@ -103,9 +108,11 @@ int main(int argc, char *argv[])
    args.AddOption(&visualization, "-vis", "--visualization", "-no-vis",
                   "--no-visualization",
                   "Enable or disable GLVis visualization.");
-   args.AddOption(&preassemble, "-pa", "--preassemble", "-mf",
-                  "--matrix-free",
-                  "Preassemble weak-divergence assuming F(u) is linear");
+   args.AddOption(&preassembleWeakDiv, "-ea", "--element-assembly-divergence",
+                  "-mf", "--matrix-free-divergence",
+                  "Weak divergence assembly level\n"
+                  "    ea - Element assembly with linearized F\n"
```

Comment:
Same remark here as in the serial version of the example: "linearized" -> "interpolated".